app/src/ui/accessibility/aria-live-container.tsx

```diff
@@ -82,6 +82,21 @@ export class AriaLiveContainer extends Component<
     )
   }
 
+  private renderMessage() {
+    // We are just using this as a typical aria-live container where the message
+    // changes per usage - no need to force re-reading of the same message.
+    if (this.props.trackedUserInput === undefined) {
```

Comment:
Found that when I added the `trackedUserInput` (or some iteration there after) that I also introduced a regression that made it so that if you did not have the `trackedUserInput` nothing was being announced. There was one other use case of this besides this component as well which is the announcing of the file status in the change file list. 

This is just saying that if the `trackedUserInput` is not provided, treat it as a regular aria live that will only re-read if the message changes. The instances where we want it reread are based on user typing so trackedUserInput would be provided.